Update calling-sdk-android.md (#70698)
Azure/azure-content: articles/communication-services/quickstarts/voice-video-calling/includes/calling-sdk-android.md

| ------------------------------------- | ------------------------------------------------------------ |
| CallClient| The CallClient is the main entry point to the Calling client library.|
| CallAgent | The CallAgent is used to start and manage calls. |
| CommunicationTokenCredential | The CommunicationTokenCredential is used as the token credential to instantiate the CallAgent.|

> | CommunicationIdentifier | The CommunicationIdentifier is used as different type of participant that would could be part of a call.|

Reviewer: ```suggestion
| CommunicationIdentifier | The CommunicationIdentifier is an object representing a call's participant.
```
